### ISSUE TITLE
Block-indentation Reports Same Value for Start and End Location

### ISSUE THREAD

**Issue by judithhinlung** [ember-template-lint/ember-template-lint#2906]
Currently, the 'block-indentation' rule is reporting the same value for start and end locations for certain indentation issues.

```
# examples
  {{#each cats as |dog|}}
       {{/each}}
```

```hbs
<LongComponentName>
  <p>Some text goes here</p></LongComponentName>
```

Sample output:
           [
             {
              "column": 8,
               "endColumn": 8,
               "endLine": 2,
               "filePath": "layout.hbs",
               "isFixable": true,
              "line": 2,
               "message": "Incorrect indentation for \`div\` beginning at L1:C0. Expected \`</div>\` ending at L2:C8 to be at an indentation of 0 but was found at 2.",
               "rule": "block-indentation",
               "severity": 2,
  {{#each cats as |dog|}}
       {{/each}}
             }
           ]

This bug could cause static analysis tools to throw an exception when using  Ember-template-lint to scan for issues.
